docs/conf.py

```diff
@@ -235,6 +236,12 @@ def linkcode_resolve(domain: str, info: Dict[str, Any]) -> Optional[str]:
     hoverxref_api_host = "/_"
 
 
+linkcheck_ignore = [
+    r"https?://github.com/.+?/.+?/(issues|pull)/\d+",
```

Comment:
why are we ignoring these?

Reply:
Ignoring these avoids around ~650 requests to Github, which are fairly unnecessary and get you ratelimited rather quickly; and the `support.discord.com` links consistently return a 403 here thanks to Cloudflare.